google-cloud-logging/lib/google/cloud/logging/v2/logging_service_v2_client.rb

```diff
@@ -71,7 +71,8 @@ class LoggingServiceV2Client
               [
                 "logName",
                 "resource",
-                "labels"
+                "labels",
+                "partialSuccess"
```

Comment:
@dazuma Are you okay with the modification to the generated GAPIC file?

Reply:
Yes I'm okay with it. There isn't any breakage at the handwritten layer.

Reply:
Ooh, I wasn't actually aware that Ruby GAPICs still supported bundling. This feature has been long deprecated in the GAPICs in languages that are designed similarly Ruby's GAPICs, and I'd really like to do the same for Ruby. I'll follow up (separate from this PR) about moving off this feature.